mode/examples/Topics/Motion/CubesWithinCube/CubesWithinCube.pyde

```diff
@@ -6,22 +6,21 @@ Collision detection against all outer cube's surfaces.
 """
 from cube import Cube
 
-halfWidth = None
-halfHeight = None
 
 Bounds = 300
 # 20 little internal cubes
 cubes = []
-for i in range(20):
```

Comment:
I disagree. ```setup()``` exists because they went with Java syntax, but Python function definitions are just statements alongside other statements. I think it's confusing to have global state spread out like that, and, whenever possible, it should be kept up at the top.